resources/assets/uad_lists.json

```
@@ -16886,7 +16886,7 @@
   {
     "id": "com.miui.smsextra",
     "list": "Oem",
-    "description": "Dependency for MIUI Messaging (MIUI SMS app misleadingly called com.android.mms)\nYou can remove it if you don't use the default SMS app (and you should)\nRun in background once the phone is booted, has access to internet and interact with Cloud Manager\n",
+    "description": "Dependency for MIUI Messaging (MIUI SMS app misleadingly called com.android.mms)\nYou can remove it if you don't use the default SMS app (and you should)\nRun in the background once the phone is booted, has access to the internet and interact with Cloud Manager.\n",
```

```suggestion
    "description": "Dependency for MIUI Messaging (MIUI SMS app misleadingly called `com.android.mms`)\nYou can remove it if you don't use the default SMS app (and you should). Run in the background once the phone is booted, has access to the internet and interact with Cloud Manager.",
```

Are you sure you mean "should" instead of "shouldn't"?

---

I Think so too.